src/main/com/fundations/convertor/model/Search.java

```diff
@@ -10,18 +10,29 @@
  * with Jala Foundation.
  */
 
-package main.com.fundations.convertor;
+package main.com.fundations.convertor.model;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  *  Search class for java applications, class which search videos
  *
  * @author Kevin Sanchez - AWT-[01].
  * @version 0.1
  */
 public class Search {
-    public Search() {
+    private List<String> lFicheros=new ArrayList<String>();
+    private String ruta;
```

Comment:
The names of variables, are in spanish. Maybe the notation should be use all in english.